src/shared/components/pipeline-run-logs/logs/LogsWrapperComponent.tsx

```diff
@@ -18,12 +18,21 @@ const LogsWrapperComponent: React.FC<LogsWrapperComponentProps> = ({ resource, .
 
   if (loaded && !error && resource.name === obj.metadata.name) {
     resourceRef.current = obj;
+  } else if (error) {
+    resourceRef.current = null;
   }
 
-  return resourceRef.current ? (
-    <MultiStreamLogs {...props} resourceName={resource.name} resource={resourceRef.current} />
-  ) : (
-    <LoadingBox />
+  let errorMessage;
+  if ((error as HttpError)?.code === 404) {
```

Comment:
```suggestion
  const errorMessage = error as HttpError)?.code === 404) ? 
```
Prefer ternary operator